src/AbstractMultivector.jl

```diff
@@ -166,20 +165,11 @@ Notes
 function getindex end
 
 """
-    inverse(M::AbstractMultivector)::AbstractMultivector
     -(M::AbstractMultivector)::AbstractMultivector
 
 Compute the additive inverse of `M`.
 """
-function inverse end
-
-"""
-    inverse(M::AbstractMultivector)::AbstractMultivector
-    -(M::AbstractMultivector)::AbstractMultivector
-
-Compute the additive inverse of `M`.
-"""
-@inline -(M::AbstractMultivector) = inverse(M)
+function - end
```

Comment:
I think we can take this line out since we are extending Base.:(-).

Reply:
I removed it, I originally thought that it would be useful for the docstring but now I realized that the built-in Base docstring is sufficient.